### PR TITLE
Fixed broken icons

### PR DESCRIPTION
**Pull request by Shivamkumar4848**

### CODE REVIEW

**Comment by mbj36** [comment]
Thanks for the PR, It will be reviewed Shortly!

(Powered by [GitMate.io](https://gitmate.io))

**Comment by luckyman20** [comment]
Squash all the commits - [https://blog.axosoft.com/git-tutorial-squashing-commits](GitKraken Tutorial for squashing commits)

File: app/views/adhocquery/adhocquerylists.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     </ul>
 
     <input ng-model="filterText" type="text" ng-keyup="onFilter()" class="span marginbottom0px form-control" placeholder="{{'label.input.filterbyname' | translate}}">
-    <a href="#/createadhocquery" class="btn btn-primary pull-right" has-permission='CREATE_ADHOC'><i class="icon-plus icon-white"></i>{{'label.anchor.createadhoc' | translate}}</a>
+    <a class="btn btn-primary pull-right ng-binding" href="#/createadhocquery" has-permission='CREATE_ADHOC'><i class="fa fa-plus fa fa-large"></i>{{'label.anchor.createadhoc' | translate}}</a>
```
Comment (gkrishnan724) [inline app/views/adhocquery/adhocquerylists.html]: Remove the `ng-binding` class.

**Comment by luckyman20** [comment]
@Shivamkumar4848 Please squash the commits

**Comment by gkrishnan724** [comment]
Thanks @Shivamkumar4848 just squash the commits

**Pull request merged**